spanner/google/cloud/spanner_v1/database.py

```diff
@@ -308,6 +309,37 @@ def batch(self):
         """
         return BatchCheckout(self)
 
+    def batch_transaction(
+            self, read_timestamp=None, min_read_timestamp=None,
+            max_staleness=None, exact_staleness=None):
```

Comment:
min_read_timestamp and max_staleness are not supported for batch transaction.

Reply:
Removed: e426c59